samples/install-without-bom/pom.xml

```diff
@@ -15,6 +15,7 @@
     <groupId>com.google.cloud.samples</groupId>
     <artifactId>shared-configuration</artifactId>
     <version>1.2.0</version>
+    <relativePath></relativePath>
```

Comment:
This change was done to mitigate below IDE error

```
'parent.relativePath' of POM com.google.cloud:spanner-snapshot:1.2.0 (/Users/arpanmishra/java-spanner/samples/snapshot/pom.xml) points at com.google.cloud:google-cloud-spanner-samples instead of com.google.cloud.samples:shared-configuration, please verify your project structure
```